deploy/charts/emqx-operator/templates/crd-emqx-plugin.yaml

```diff
@@ -1,4 +1,5 @@
 {{- if .Values.installCRDs }}
+
```

Comment:
It looks like this file doesn't need to be updated

Reply:
It looks like the file hasn't been changed, it's just been formatted

Reply:
rollback